### ISSUE TITLE
[Request] Add support for HomeDIRECT Beko devices...

### ISSUE THREAD

**Issue by rafpigna** [home-assistant-HomeWhiz/home-assistant-HomeWhiz#231]
Hello
I hope this is the right place to ask, because I didnt find another place where ask this. If not, please help me to understand where to post this request.

I bought a Beko AC with Wifi Support, choosing this brand because I saw that this integration for my home assistant installation.

But at the end of plumbing and electrical jobs, I discovered that the AC unit now is using ANOTHER APP called "Home Direct", no more "HomeWiz".
Basically both are the same app, also the logo is the same, but newest devices are not supported in the Home Wiz app, and you have to use Home Direct.
At least, this is what I understood.
I already tried to add my AC to the "old" Home Wiz app but without success.

My question: 
There is some plan to support the HomeDirect devices in the future? 
There is another way that I dont know to add the device to my Home Assistant?

Thanks

**Comment by TechHummel** [comment]
Hello @rafpigna 👋 
I'm not planing to add any HomeDirect devices to this integration and also don't know how to add them to Home Assistant - I'm sorry to disappoint you 😞 
I don't own a single HomeDirect device, which would make supporting such devices very difficult.

**Comment by mbriacuni** [comment]
@rafpigna 
I had the same problem, Beko air conditioner recently purchased, especially for integration on Home Assistant and once mounted I discovered that in reality the app it uses is not HomeWhiz, but HomeDirect, and therefore I could not integrate it. Coincidentally I discovered that the air conditioner in a network scan of the Fing app was recognized as a "Tuya device" and in fact trying to add it on Smart Life (compatible with HA) everything works regularly!!!

**Comment by rafpigna** [comment]
> @rafpigna I had the same problem, Beko air conditioner recently purchased, especially for integration on Home Assistant and once mounted I discovered that in reality the app it uses is not HomeWhiz, but HomeDirect, and therefore I could not integrate it. Coincidentally I discovered that the air conditioner in a network scan of the Fing app was recognized as a "Tuya device" and in fact trying to add it on Smart Life (compatible with HA) everything works regularly!!!

First of all thanks for your reply.
I didn't understood... Did you added it in the Fing app and then the Fing app in HA?
Or did you add it to Smart Life app?

**Comment by mbriacuni** [comment]
> > @rafpigna I had the same problem, Beko air conditioner recently purchased, especially for integration on Home Assistant and once mounted I discovered that in reality the app it uses is not HomeWhiz, but HomeDirect, and therefore I could not integrate it. Coincidentally I discovered that the air conditioner in a network scan of the Fing app was recognized as a "Tuya device" and in fact trying to add it on Smart Life (compatible with HA) everything works regularly!!!
> 
> First of all thanks for your reply. I didn't understood... Did you added it in the Fing app and then the Fing app in HA? Or did you add it to Smart Life app?

No, Fing is an app that scans the devices connected to the local network to get some more info than what you would see in the router control panel.

I added the air conditioner on Smart Life app.

**Comment by giannitw** [comment]
> [@rafpigna](https://github.com/rafpigna) I had the same problem, Beko air conditioner recently purchased, especially for integration on Home Assistant and once mounted I discovered that in reality the app it uses is not HomeWhiz, but HomeDirect, and therefore I could not integrate it. Coincidentally I discovered that the air conditioner in a network scan of the Fing app was recognized as a "Tuya device" and in fact trying to add it on Smart Life (compatible with HA) everything works regularly!!!

**Comment by tottelitott** [comment]
Had a chat with Grundig - My Grundig is not for german market, it's for italy market :(

So Homewhiz is only for the more solvent customers, homedirect seems to be for the poorer people! I managed to get a working product code for homewhiz but it fails when i want to add the device (wifi name is homedirect123 and not homewhiz123) it can't connect because of the wrong wifi name... 

This absolutely sucks because there is no integration for homedirect. The app is also garbage - air condition page is all in chinese...

OK, i'll replace this grundig garbage next year with a panasonic... This integration works fine.

**Comment by OlegIshchenko30** [comment]
> > > [@rafpigna](https://github.com/rafpigna) I had the same problem, Beko air conditioner recently purchased, especially for integration on Home Assistant and once mounted I discovered that in reality the app it uses is not HomeWhiz, but HomeDirect, and therefore I could not integrate it. Coincidentally I discovered that the air conditioner in a network scan of the Fing app was recognized as a "Tuya device" and in fact trying to add it on Smart Life (compatible with HA) everything works regularly!!!
> > 
> > 
> > First of all thanks for your reply. I didn't understood... Did you added it in the Fing app and then the Fing app in HA? Or did you add it to Smart Life app?
> 
> No, Fing is an app that scans the devices connected to the local network to get some more info than what you would see in the router control panel.
> 
> I added the air conditioner on Smart Life app.

Thanks a lot. It solved my problem.
